### ISSUE TITLE
Iron Banner Armor not Filtering

### ISSUE THREAD

**Issue by rvvisse** [dasilva333/TowerGhostForDestiny#97]
I have Hunter gauntlets, Titan boots, Hunter cloak, Titan Mark and the emblem.  Only the cloak appears with the Iron Banner filter selected. 

I'm running the latest build on the iPhone 6+ (4/26/15). The version number was recently removed from the title bar and I can't seem to locate it elsewhere to include it.

**Comment by dasilva333** [comment]
I'm changing versions around, I'm aware of the issue with the Sets feature not including pre-TDB items, so if you had lvl 30 IB gear it will only recognize lvl 32 IB gear basically I'll mark this as an enhancement to improve in the future. Thanks

**Comment by rvvisse** [comment]
Thanks!!  I'm thinking about that full lvl 34 IB set when HoW comes out. I appreciate the reply!

-Rick

**Comment by M3Rocket** [comment]
Currently using 2.3.7. I just acquired IB level 32 Titan Mark, Helmet and Chest this past week. But when the IB Armor filter is on, it does not show that I have any of those items. When I select Show Missing Items, it doesn't show the post-TDB Titan Helmet or Chest piece (indicated by the "I" on the top right of the icon), but shows the Titan Mark.

**Comment by dn4cer** [comment]
None of my Titan's IB gear is being recognized by the "Sets" filter.

Iron Regalia Great Helm
Iron Regalia Plate
Iron Regalia Greaves
Jolder's Iron Sash

I don't have the gauntlets so I don't know if those would get detected or not.

**Comment by TopTenTiger** [comment]
My Iron Regalia Gauntlets (post-TDB) are not recognized by the Iron Banner Armour Filter.

**Comment by dasilva333** [comment]
Hey guys so to bring everyone up to speed on why this is an issue, the code is from DestinyExotics, it's hardcoded to recognize only one version of the item, I came up with a temporary fix as a sibling structure that provides the ids that it's missing. I filled in some of the Iron Banner Weapons but haven't managed to catch them all. The bulk of the work is simply gathering the unique id of these items and putting them in that structure. 

https://github.com/dasilva333/TowerGhostForDestiny/blob/master/www/js/destinyexotics.js#L150

The way I did it was by visiting PlanetDestiny and filtering for iron banner like so:
http://db.planetdestiny.com/items/weapons?q=iron%20banner

If anyone would want to lend a hand and just fill in the rest of these blocks with the rest of the information that would be great. Thanks guys
